addons/point_of_sale/static/src/js/db.js

```diff
@@ -241,7 +241,7 @@ var PosDB = core.Class.extend({
     },
     add_partners: function(partners){
         var updated_count = 0;
-        var new_write_date = '';
+        var new_write_date = new Date().toISOString().replace(/(.*T.*)(.*Z.*)$/, '');
```

Comment:
Can you fully explain the motivation to this change? In my opinion, it is not needed because the goal is to have the `partner_write_date` to be the latest date _among_ the partners being added (which came from the server). If we set it to the current time, there is a _chance_[1] that the partner modified from different clients won't be fetched as the new `product_write_date` can become higher than the one modified.

[1] Depends on the speed of the network communication between client and the server.